Message.php

```diff
@@ -28,6 +28,7 @@ class Message extends BaseMessage
      * @var \Swift_Message Swift message instance.
      */
     private $_swiftMessage;
+    public $messageClass = "\Swift_Message";
```

Comment:
messageClass is al ready reserved for Mailer. this variable is set on Message and refers to the Swift_Message